src/Blocks/ChunkInterface.cpp

```diff
@@ -1,13 +1,12 @@
 
 #include "Globals.h"
 
-#include "ChunkInterface.h"
 #include "BlockHandler.h"
 #include "WorldInterface.h"
 #include "../ChunkMap.h"
 #include "../World.h"
 #include "../BlockInfo.h"
-
+#include "ChunkInterface.h"
```

Comment:
If it fails to compile with the include at the top, then there are includes missing from the "ChunkInterface.h" header